tests/helpers.test.ts

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Neil Enns. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import {closeSync, existsSync, openSync, unlinkSync, writeFileSync} from 'fs';
+import * as JSONC from "jsonc-parser";
+import * as helpers from "./../src/helpers";
+
+describe("helpers", () => {
+  const settingsFilePath = `${__dirname}/settings.json`;
+
+  beforeEach(() => {
+    closeSync(openSync(settingsFilePath, 'w'));
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    if (existsSync(settingsFilePath)) {
+      unlinkSync(settingsFilePath);
+    }
+  });
+
+   test("Verify can load settings.json", () => {
+     const serviceName = "Settings";
+     const expectedSettings = {"foo": "bar"};
+     writeFileSync(settingsFilePath, JSON.stringify(expectedSettings));
+
+     const actualSettings = helpers.readSettings(serviceName, settingsFilePath);
+
+     expect(actualSettings).toEqual(expectedSettings);
+   });
+
+   test("Verify cannot load settings.json because it does not exist", () => {
+     //eslint-disable-next-line no-console
+     console.log = jest.fn();
+     const serviceName = "Settings";
+     unlinkSync(settingsFilePath);
+
+     const actualSettings = helpers.readSettings(serviceName, settingsFilePath);
+
+     //eslint-disable-next-line no-console
+     expect(console.log).toHaveBeenCalledWith(expect.stringContaining("[Settings] Unable to read the configuration file: ENOENT: no such file or directory"));
+     expect(actualSettings).toBeNull();
+   });
+
+   test("Verify throws if rawConfig empty", () => {
+     const serviceName = "Settings";
+     const expectedSettings = "";
+     writeFileSync(settingsFilePath, expectedSettings);
+
+     expect(() => {helpers.readSettings(serviceName, settingsFilePath)}).toThrow(Error);
+   });
+
+   test("Verify throws with message if rawConfig empty", () => {
+     const serviceName = "Settings";
+     const expectedSettings = "";
+     writeFileSync(settingsFilePath, expectedSettings);
+
+     try {
+       helpers.readSettings(serviceName, settingsFilePath);
+     } catch (error) {
+       expect(error.message).toBe(`[${serviceName}] Unable to load configuration file ${settingsFilePath}.`);
+     }
+   });
```

Comment:
To clarify a bit here. I write two tests, where one makes for sure that the function threw and the other tests the error message. If I deleted the first of the two and left the second, it's possible that the function does not throw and it goes unnoticed because it wouldn't go into the `catch` and no assertions is considered a successful test, making it a false-positive.